public/locale/en.json

```
@@ -2097,6 +2097,7 @@
   "permanent_address": "Permanent Address",
   "permission": "Permission",
   "permission_denied": "You do not have permission to perform this action",
+  "permission_denied_encounter": "You do not have permission to view this encounter",
```

not a big concern, but i'd prefer having:

```suggestion
  "permission_denied_for_action": "You do not have permission to perform {{ action }}",
  "permission_denied_for_view_entity": "You do not have permission to view this {{ entity }}",
```

---

Wouldn't work with another languages as well 🤔